packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -250,7 +263,7 @@ export class TerminalFrontendContribution implements FrontendApplicationContribu
             this.storageService.getData<string>(ENVIRONMENT_VARIABLE_COLLECTIONS_KEY).then(data => {
                 if (data) {
                     const collectionsJson: SerializableExtensionEnvironmentVariableCollection[] = JSON.parse(data);
-                    collectionsJson.forEach(c => this.shellTerminalServer.setCollection(c.extensionIdentifier, true, c.collection));
+                    collectionsJson.forEach(c => this.shellTerminalServer.setCollection(c.extensionIdentifier, true, c.collection, undefined));
```

Comment:
If we make the description part of the env var collection, we can serialize it at the same time, right?

Reply:
I've added the description to `SerializableExtensionEnvironmentVariableCollection` now.